manifests/init.pp

```diff
@@ -138,7 +138,7 @@
       vcsrepo { "${basedir}/puppetboard":
         ensure   => present,
         provider => git,
-        owner    => $user,
+        user     => $user,
```

Comment:
will that break existing installations? was that maybe setup like that on purpose, so the user that runs it has no permissions / less permissions?

Reply:
The only case I can see is if the user cannot traverse one of the parent directories.

Because of https://github.com/voxpupuli/puppet-puppetboard/pull/349#pullrequestreview-872397712 I would also want to change this user to root:root by default, but I _this_ will be a breaking change